Drivers/Src/CANbus.c

```diff
@@ -121,6 +179,20 @@ ErrorStatus CANbus_BlockAndSend(CANId_t id, CANPayload_t payload) {
 	return result;
 }
 
+/**
+ * @brief   Transmits data onto the CANbus without mailbox semaphores.
+ * @param   id : CAN id of the message
+ * @param   payload : the data that will be sent.
+ * @return  ERROR if error, SUCCESS otherwise
+ */
+ErrorStatus CANbus_BlockAndSend_FaultState(CANId_t id, CANPayload_t payload) {
```

Comment:
this function seems kinda redundant but I guess we can keep it for interface consistency.
you should make this a one liner though, no need for the extra code and extra variables:
```return CANbus_SendMsg_FaultState(id, payload);```

Reply:
wait yea, why don't you just make it one function instead of 2? I don't see a reason to have the other one be its own since nothing else will call it (and it is blocking).

Reply:
fixed how tienda suggested. the function still exists but it is a one liner

Reply:
Either remove the static function and place the logic in this function, or remove static from the above function. I see no point in having a function call a static function that isn't used in any other part of the canbus library unless you have a reason for this.

Reply:
Removed Static

Reply:
I don't think you understand what I'm saying. I don't think it is necessary to have a function call another function, so remove one of them. like delete it.